### PR TITLE
Add vote announcement to voting procedure

### PR DESCRIPTION
**Pull request by t8m**
Simple adjustment of the voting procedure to announce votes via openssl-project ML.

### CODE REVIEW

File: policies/voting-procedure.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ pull request of the policy change proposal.
 
 Any other votes are recorded as separate issues in the repository.
 
+All votes governed by this procedure should be announced through an
```
Comment (romen) [inline policies/voting-procedure.md:20]: s/should/must/

File: policies/voting-procedure.md [hunk 2]
```diff
@@ -17,6 +17,10 @@ pull request of the policy change proposal.
...
+e-mail to the
+[OpenSSL Project mailing list](https://mta.openssl.org/mailman/listinfo/openssl-project).
+
```
Comment (romen) [inline policies/voting-procedure.md:23]: Add something similar to:

The announcement email must contain a hyperlink to the GitHub issue where the vote is carried out.

**Review by romen — CHANGES_REQUESTED** [review]
Collecting some ideas that came up during the OTC discussion of this PR

**Comment by t8m** [comment]
Fixup pushed addressing the OTC comments.

**Comment by romen** [comment]
Last week we did not have consensus about one last item: when closing the vote, should it be also announced on openssl-project?

I'd prefer if we did that.

If it is too much of a burden, should we start looking more concretely into the automation ideas we discussed when we started moving the voting procedures to GitHub?

**Comment by t8m** [comment]
I feel like there are already too many steps you have to manually do for all the votes. And we already have an internal issue to send notifications for merges to technical-policies and general-policies repos to go into the openssl-project ML.

**Comment by mspncp** [comment]
Vote: [+1]

**Comment by paulidale** [comment]
Vote: [+1]

**Comment by t-j-h** [comment]
Vote: [-1]

Anyone interested in the discussions and decisions around policies should be interacting with the details on github.com and approved policies are on the public website (as links). There is no point in a separate announcement mechanism.

Adding more traffic to openssl-project is not the right move IMHO and in fact there should be less email traffic in general.

**Comment by beldmit** [comment]
Vote: [-1]

The less manual steps we have, the better are the procedures.

**Comment by slontis** [comment]
Vote: [+1]

**Comment by t8m** [comment]
@levitte @kroeckx ping for votes

**Comment by kroeckx** [comment]
Voting +1

**Comment by levitte** [comment]
Vote: [+1]

I think that at least for the time being, we need to do this...  Ideally, it would be nice if we only did this on github, but I don't think that everyone is up to speed yet.  This should be revisited in a year or so, and if we decide to change this procedure, it should be announced in the project list some time prior.

**Comment by t8m** [comment]
Closing the vote.
```
Topic: Add vote announcement to voting procedure at commit 0737de7
       This will become an official OTC policy.
Proposed by: Tomas
Issue link: https://github.com/openssl/technical-policies/pull/45
Public: yes
Opened: 2022-05-09
Closed: 2022-05-18
Accepted:yes    (for: 8, against: 2, abstained: 0, not voted: 0)

   Dmitry     [-1]
   Matt       [+1]
   Pauli      [+1]
   Tim        [-1]
   Richard    [+1]
   Shane      [+1]
   Tomas      [+1]
   Kurt       [+1]
   Matthias   [+1]
   Nicola     [+1]
```

**Comment by t8m** [comment]
Merged and vote recorded.

**Pull request closed without merging**